packages/cadl-typescript/src/modelUtils.ts

```diff
@@ -334,10 +335,13 @@ function getSchemaForModel(
   usage?: SchemaContext[],
   needRef?: boolean
 ) {
-  const friendlyName = getFriendlyName(program, model);
+  const overridedModelName =
+    getProjectedName(program, model, "javascript") ??
+    getProjectedName(program, model, "client") ??
+    getFriendlyName(program, model);
```

Comment:
I forgot, what is a friendlyName?

Reply:
https://microsoft.github.io/typespec/standard-library/built-in-decorators#friendlyname

![image](https://user-images.githubusercontent.com/9943211/224648801-5b2de819-87de-4fd8-8e48-d862d052068d.png)